atox/src/main/res/layout/fragment_call.xml

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:custom="http://schemas.android.com/apk/res-auto"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:orientation="vertical">
-    <ImageView android:id="@+id/call_background"
-        android:layout_width="fill_parent"
-        android:layout_height="fill_parent"
-        android:adjustViewBounds="true"
-        android:importantForAccessibility="no"
-        android:scaleType="centerCrop" />
+
+    <ltd.evilcorp.atox.ui.AvatarImageView
```

Comment:
Not sure this is a good change. The idea was to use the call background as the video container as well, and this looks a bit awkward the way it's positioned/sized after this. Maybe it'll look better a bit bigger and with `layout_above="@id/control_container" `?

Reply:
It snaps it right above the container this way. And what is the `ShapeableImageView` called `user_avatar`? I didn't notice that.

Reply:
Ah, I was hoping that + centerInParent would lead to it being centered in the area above the control_container. Right now it's a bit too far down. :(

The user_avatar is a placeholder thing. It was located where I was going to show the user a preview of the outgoing video, but it's not used right now since I decided not to work on video for 0.7.0. :P

Reply:
It can easily fixed by adding another `RelativeLayout` for the `AvatarImageView`

Reply:
If I increase the size, there's a chance of it being cropped in small devices in landscape mode. Should I still increase it and set a dynamic size in the UI redesign of this screen?

Reply:
Or set a dynamic size with a different PR regardless?

Reply:
Probably update to a dynamic size in this PR since it was changed from filling the entire background to being fixed-size in the center in this PR?

Reply:
OK. But doing that after a commit in which I switch to AvatarImageView and set a fixed size right?